app/helpers/reactjs_helper.rb

```diff
@@ -55,15 +55,21 @@ def select_requested_plugins(plugin_names)
 
   def js_tags_for(requested_plugins)
     requested_plugins.map do |plugin|
-      name = plugin.to_s.tr('-', '_')
+      name = plugin.normalized_id
       javascript_tag("window.tfm.tools.loadPluginModule('/webpack/#{name}','#{name}','./index');".html_safe)
     end
   end
 
   def global_js_tags(requested_plugins)
```

Comment:
I think this should really have been a private helper. Perhaps some other methods too.